models/issues/assignees.go

```diff
@@ -35,6 +39,7 @@ func (issue *Issue) LoadAssignees(ctx context.Context) (err error) {
 		Join("INNER", "issue_assignees", "assignee_id = `user`.id").
 		Where("issue_assignees.issue_id = ?", issue.ID).
 		Find(&issue.Assignees)
+	issue.isAssigneeLoaded = true
```

Comment:
IMO, if error occurs, we should not set this flag as true.

Reply:
bb45ecb70754c0f5530e809a42b2a678b68299a1

Reply:
not only here, but also other places

Reply:
e1ee0aa